config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -288,6 +288,32 @@ public ServerHttpSecurity addFilterAt(WebFilter webFilter, SecurityWebFiltersOrd
 		return this;
 	}
 
+	/**
+	 * Adds a {@link WebFilter} before specific position.
+	 * @param webFilter the {@link WebFilter} to add
+	 * @param order the place before which to insert the {@link WebFilter}
+	 * @return the {@link ServerHttpSecurity} to continue configuring
+	 * @since 5.2.0
+	 * @author Rob Winch
```

Comment:
Can you either remove my name or replace with yours?

Reply:
Author changed to me!!